test/controllers/publishers/site_banners_controller_test.rb

```diff
@@ -38,27 +38,31 @@ class PublishersControllerTest < ActionDispatch::IntegrationTest
     site_banner = site_banners(:default)
     sign_in publisher
 
-    fake_data = "A" * Publishers::SiteBannersController::MAX_IMAGE_SIZE
+    fake_data = "A" * SiteBanner::LOGO_UNIVERSAL_FILE_SIZE
```

Comment:
```suggestion
    fake_data = "A" * Publishers::SiteBannersController::MAX_IMAGE_SIZE
```

Reply:
This shouldn't the test's result, but this matches what the controller is checking

Reply:
Fixed.